helm-charts/common/ollama-service/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+{{- if .Values.global.monitoring }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "ollama-service.fullname" . }}
+  labels:
+    release: {{ .Values.global.prometheusRelease }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "ollama-service.selectorLabels" . | nindent 6 }}
+  endpoints:
+  - port: ollama-service
+    interval: 5s
+{{- end }}
```

Comment:
According to this ticket, Ollama does not provide Prometheus `/metrics` endpoint yet, so this is redundant:
https://github.com/ollama/ollama/issues/3144